PyDSS/dssInstance.py

```diff
@@ -110,13 +110,19 @@ def __init__(self, params):
         self._CreateBusObjects()
         self._dssSolver.reSolve()
 
+
         if params['Profiles']["Use profile manager"]:
             #TODO: disable internal profiles
             self._Logger.info('Disabling internal yearly and duty-cycle profiles.')
             for m in ["Loads", "PVSystem", "Generator", "Storage"]:
                 run_command(f'BatchEdit {m}..* yearly=NONE duty=None')
-            self.profileStore = ProfileManager(self._dssObjects, self._dssSolver, params)
-            self.profileStore.setup_profiles()
+            profileSettings = self._Options["Profiles"]["settings"]
+            profileSettings["objects"] = self._dssObjects
+            self.profileStore = ProfileInterface.Create(
+                self._dssInstance, self._dssSolver, self._Options, self._Logger, **profileSettings
```

Comment:
Can we stop passing `dssInstance` to other classes and modules? Those other modules can `import opendssdirect as dss` if they need to use it. This would be much clearer to new users. Some people might get confused as to what is an OpenDSS thing and what is a PyDSS thing.

Reply:
Lets us clean hus up after  the merge